blog/rosi_bajari/index.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: category
+title: Rosi Posts
+category: rosi_bajari
+permalink: /blog/category/rosi_bajari
+header: 
+    version: small
+    title: Technical Business Analyst
+    image: header-logo-crop.png
+    page: team
+    icon: icon-blog
+---
```

Comment:
@rosibaj I added this for you :) it'll populate this page: https://softeng.oicr.on.ca/blog/category/rosi_bajari

Hope that works :)

Reply:
thanks minh thats prefect!